.travis.yml

```diff
@@ -27,24 +27,40 @@ jobs:
 
     # Primary platforms
 
+    # pod lib lint to check build and warnings
     - stage: test
       env:
         - PROJECT=Firebase PLATFORM=iOS
```

Comment:
Should this also have `METHOD=pod-lib-lint`?

Reply:
Yes thanks.